src/mem/ruby/protocol/GPU_VIPER-TCC.sm

```diff
@@ -717,18 +746,21 @@ machine(MachineType:TCC, "TCC Cache")
       // woken up
       st_stallAndWaitRequest;
   }
-  transition(IV, {WrVicBlk, Atomic, WrVicBlkBack}) { //TagArrayRead} {
+
+  transition(IV, {WrVicBlk, Atomic, AtomicPassOn, WrVicBlkBack}) { //TagArrayRead} {
```

Comment:
Here too, don't we want TagArrayRead for them?

Reply:
Removed as extraneous per @abmerop's comment.